### ISSUE TITLE
Trying to reproduce human Semantic Parsing in pytorch

### ISSUE THREAD

**Issue by TJJTJJTJJ** [emrahbasaran/SPReID#16]
Hello, I am a Postgraduate in Beijing University in China. I am trying to reproduce human Semantic parsing in pytorch, I have seen your code in your release code. In my code, I have reproduce the netword in human semantic parsing , which is same as yours human semantic parsing in pytorch. and I use interplot 30*30-----> 512*512 and cross_entropy to measure loss between results and ground label，when I train in LIP, its results is so bad. I have seen the LIP paper to learn the ASPP. And I have tried to load your released model parameter in  LIP_iter_30000.chainermodel to pytorch. But it doesn't work, the results is so bad, I can't see any segmentation in picture. So I think maybe there is something wrong about my code. So if you are free, please tell how  to train the semantic parsing. 
My email: 17120413@bjtu.edu.cn .

**Comment by upgirlnana** [comment]
> Hello, I am a Postgraduate in Beijing University in China. I am trying to reproduce human Semantic parsing in pytorch, I have seen your code in your release code. In my code, I have reproduce the netword in human semantic parsing , which is same as yours human semantic parsing in pytorch. and I use interplot 30_30-----> 512_512 and cross_entropy to measure loss between results and ground label，when I train in LIP, its results is so bad. I have seen the LIP paper to learn the ASPP. And I have tried to load your released model parameter in LIP_iter_30000.chainermodel to pytorch. But it doesn't work, the results is so bad, I can't see any segmentation in picture. So I think maybe there is something wrong about my code. So if you are free, please tell how to train the semantic parsing.
> My email: [17120413@bjtu.edu.cn](mailto:17120413@bjtu.edu.cn) .
i wonder if you have solved the problem of pytorch training of semantic parsing. 
 I have a similar problem, I wonder if you can answer this question and share your pytorch code.​

**Comment by upnana** [comment]
@TJJTJJTJJ can you share your pyotrch code with me

**Comment by TJJTJJTJJ** [comment]
sorry, the results are too bad, so I have deleted the code I have written.

**Comment by TJJTJJTJJ** [comment]
> @TJJTJJTJJ can you share your pyotrch code with me
@upgirlnana 
sorry, the results are too bad, so I have deleted the code I have written.

**Comment by ajwl-pmli** [comment]
Hello, Do you know how to compute the rank1 and mAP? could you give me the code which compute the rank1 and mAP, My mail is pku1401210454@163.com, Thank you very much! @TJJTJJTJJ @upgirlnana @ebsrn
